app/containers/App.jsx

```diff
@@ -16,18 +14,15 @@ const cx = classNames.bind(styles);
  * A better explanation of react-router is available here:
  * https://github.com/rackt/react-router/blob/latest/docs/Introduction.md
  */
-const App = ({children}) => {
-  return (
-    <div className={cx('app')}>
-      <Navigation />
-      <Message />
-        {children}
-    </div>
-  );
-};
+const App = ({children}) => (
+  <div className={cx('app')}>
```

Comment:
@psimyn @jrodl3r @caranicas do you guys like this bit of coupling to `classnames?`

Perhaps this should suffice?
```javascript
const { app }  = styles;

...
className={app}
```

I'm definitely a fan of `classNames` and see benefit in using it, but as a minimal boilerplate it might be a bit too much and gives that bit of overhead in understanding how this app works.